source/cpu/cpuid_linux_s390x.go

```diff
@@ -24,6 +24,12 @@ unsigned long gethwcap() {
 }
 */
 import "C"
+import (
```

Comment:
Could you add a new line between your imports and `import "C"`?

Reply:
Done.